de4a-connector/src/main/java/eu/de4a/connector/client/Client.java

```diff
@@ -86,7 +88,7 @@ public NodeInfo getNodeInfo(String participantId, String documentTypeId, boolean
 		try {
 			// Requires the form iso6523-actorid-upis::9915:demo
 			final IParticipantIdentifier aPI = SimpleIdentifierFactory.INSTANCE
-					.parseParticipantIdentifier(participantId);
+					.parseParticipantIdentifier(participantId.toLowerCase(Locale.getDefault()));
```

Comment:
Is this the same as "Locale.ROOT"?  I prefer the other one (also because it is a constant)